projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -910,7 +910,7 @@ public static function parse( $attributes, $content, $context = array() ) {
 			'error_types'    => array(
 				'is_required'        => __( 'This field is required.', 'jetpack-forms' ),
 				'invalid_form_empty' => __( 'The form you are trying to submit is empty.', 'jetpack-forms' ),
-				'invalid_form'       => __( 'Please fill out the form correctly.', 'jetpack-forms' ),
+				'invalid_form'       => __( 'Please complete all required fields to continue.', 'jetpack-forms' ),
```

Comment:
We might need to split this error into two; one for multiple required fields, missing and another for invalid content like invalid email.

You might have no required fields at all in the form, and only issue is that email was filled incorrect, for example.